code/game/objects/items/inspector.dm

```diff
@@ -91,6 +91,23 @@
 	else
 		. += "\The [cell] is firmly in place. [span_info("Ctrl-click with an empty hand to remove it.")]"
 
+/obj/item/inspector/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
```

Comment:
Don't use afterattack anymore, use `interact_with_atom` and/or `ranged_interact_with_atom`